### DISCUSSION TITLE
Eval json does not match Coco results

### DISCUSSION THREAD

**Discussion by HoopsMcann** [obss/sahi#657] [Q&A] [answered]
Here is the coco results from running `sahi evaluate`:

` Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=500 ] = 0.506
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=500 ] = 0.780
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=500 ] = 0.531
 Average Precision  (AP) @[ IoU=0.50      | area= small | maxDets=500 ] = 0.723
 Average Precision  (AP) @[ IoU=0.50      | area=medium | maxDets=500 ] = 0.594
 Average Precision  (AP) @[ IoU=0.50      | area= large | maxDets=500 ] = 0.890
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=500 ] = 0.357
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=500 ] = 0.295
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=500 ] = 0.627
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=500 ] = 0.439
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=500 ] = 0.439
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=500 ] = 0.723
COCO evaluation results are successfully exported to vision/doc_inference/eval.json`

And here are the results that are written to the `eval.json`:
`{
    "bbox_mAP": 0.506,
    "bbox_mAP50": 0.78,
    "bbox_mAP75": 0.531,
    "bbox_mAP_s": 0.357,
    "bbox_mAP_m": 0.295,
    "bbox_mAP_l": 0.627,
    "bbox_mAP50_s": 0.723,
    "bbox_mAP50_m": 0.594,
    "bbox_mAP50_l": 0.89,
    "bbox_mAP_copypaste": "0.506 0.531 0.780 0.357 0.295 0.627 0.723 0.594 0.890"
}`

All the keys in the eval.json file show precision metrics, but no recall metrics.

**Comment by fcakyon** [comment, marked as answer]
Hello @HoopsMcann, currently recall metrics are not exported into eval json, you can open a PR if you need that feature :+1:
